lib/SPIRV/libSPIRV/SPIRVMemAliasingINTEL.h

```diff
@@ -0,0 +1,76 @@
+//===- SPIRVMemAliasingINTEL.h -                               --*- C++ -*-===//
+//
+//                     The LLVM/SPIRV Translator
+//
+// This file is distributed under the University of Illinois Open Source
+// License. See LICENSE.TXT for details.
```

Comment:
@MrSidims please double check that this is the right license.

Reply:
TBH, I've taken it from SPIRVAsm.h :
```
  1 //===- SPIRVAsm.h -                                           --*- C++ -*-===//
  2 //
  3 //                     The LLVM/SPIRV Translator
  4 //
  5 // This file is distributed under the University of Illinois Open Source
  6 // License. See LICENSE.TXT for details.
  7 //
  8 //===----------------------------------------------------------------------===//
```

I'll double check it.